.template/addons/github/.github/workflows/review_code_using_danger.yml.tt

```diff
@@ -0,0 +1,57 @@
+name: Automated code review
```

Comment:
@longnd Can the file name also be automated_code_review ? 🤔

Reply:
I renamed the file to `review_code` in e4ccfdd to match our [naming convention](https://nimblehq.co/compass/development/code-conventions/github-actions/#naming) that requires 
> Name workflow files following the pattern `verb_environment.yml`